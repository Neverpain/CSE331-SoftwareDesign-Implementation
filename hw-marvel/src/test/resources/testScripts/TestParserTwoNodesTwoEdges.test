# Tests that the makeGraph method properly creates the nodes with edges
# going both directions, prints out path between both

LoadGraph g twoNodeGraph.csv

FindPath g n1 n2

FindPath g n2 n1