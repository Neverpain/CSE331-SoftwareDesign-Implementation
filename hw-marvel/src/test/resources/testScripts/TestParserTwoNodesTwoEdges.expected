# Tests that the makeGraph method properly creates the nodes with edges
# going both directions, prints out path between both

loaded graph g

path from n1 to n2:
n1 to n2 via same

path from n2 to n1:
n2 to n1 via same